### PR TITLE
M3-5471: Add reference to Gravatar to My Profile settings

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

This PR adds a new section to the Display tab under My Profile to encourage users to using individual profile photos rather than the default user icon. 

Some other UI changes to the Display tab include:
- Moving the other sections into the same `<Paper />` component
- Adding a label and descriptive text to the Timezone selector
- Aligning the buttons to the right to reduce vertical spacing
- Changing the button text to better explain what fields it corresponds to

## How to test

- Go to `/profile/display`
- Create a Gravatar account to update existing profile photo or change the existing photo via the links provided
- Check UI for restricted users

**How do I run relevant unit tests?**

- `yarn test SingleTextFieldForm`
- `yarn test TimezoneForm`

### CODE REVIEW

**Review by HanaXu — COMMENTED** [review]
Can we look into using React Query for getting the Gravatar url so that the url will be cached since other components will also need to use the Gravatar url?

**Review by DevDW — APPROVED** [review]

**Comment by tiffwong** [comment]
@HanaXu I think that's out of scope for this PR but I can create a follow-up ticket to address that

File: packages/manager/src/features/Profile/DisplaySettings/DisplaySettings.tsx [hunk 1]
```diff
@@ -68,56 +142,105 @@ export const DisplaySettings: React.FC<WithNotifications> = (props) => {
...
+        gravatarURL === undefined ? (
+          <div className={classes.avatar}>
+            <UserIcon />
+          </div>
+        ) : (
+          <div className={classes.avatar}>
+            <img
+              className={classes.gravatar}
+              src={gravatarURL}
+              alt="Gravatar"
```
Comment (bnussman) [inline packages/manager/src/features/Profile/DisplaySettings/DisplaySettings.tsx:171]: The image should be rounded

Screen Shot 2022-03-30 at 10 01 49 AM

**Review by bnussman — APPROVED** [review]
This looks awesome! 🚀

**Pull request merged**